openfisca_us_data/datasets/acs/raw_acs.py

```diff
@@ -2,64 +2,37 @@
 import requests
 from io import BytesIO
 import pandas as pd
-
+from zipfile import ZipFile
 
 @dataset
 class RawACS:
     name = "raw_acs"
 
     def generate(year: int) -> None:
-        url = f"https://www2.census.gov/programs-surveys/supplemental-poverty-measure/datasets/spm/spm_{year}_pu.dta"
+        url = f"https://www2.census.gov/programs-surveys/acs/data/pums/{year}/1-Year/csv_pus.zip"
+        request = requests.get(url)
+        file = ZipFile(BytesIO(request.content))
+        file.extractall(f'{year}_pus')
```

Comment:
@nikhilwoodruff feel free to suggest otherwise given these are large files (larger than others), but to be consistent with other `generate` functions I think we'll want to avoid writing the source files to disk, and instead load from the zip file directly. This might make most sense as a function, something like this (not sure if it'll work):
```python
def concat_zipped_csvs(url: str, prefix: str):
    zf = ZipFile(BytesIO(requests.get(url)))
    a = pd.read_csv(zf.open(prefix + "a.csv"))
    b = pd.read_csv(zf.open(prefix + "b.csv"))
    res = pd.concat([a, b]).fillna(0)
    res.columns = res.columns.str.lower()
    return res
```
Then called as:
```python
person_df = concat_zipped_csvs(
    f"https://www2.census.gov/programs-surveys/acs/data/pums/{year}/1-Year/csv_pus.zip",
    "psam_pus"
)
```
And similarly for household.

Reply:
I think that's fine as a general approach for large files - though I did envisage something similar to `openfisca-uk-data`'s `download()` function (either from gcp or GitHub) being useful hre